### ISSUE TITLE
Documentation Link Request

### ISSUE THREAD

**Issue by frioux** [pplu/aws-sdk-perl#85]
I haven't fully dug through this yet, but I suspect that the CloudWatch docs could link to http://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/CW_Support_For_AWS.html liberally, especially the GetMetricStatistics and probably Dimension docs.

I'd send a Pull Request but I feel like I don't know enough AWS or Paws to reasonably do so.

**Comment by spazm** [comment]
opened an upstream ticket with boto: https://github.com/boto/botocore/issues/1279. 

Boto does have documentation links, but they use a redirection url that doesn't work consistently

* fails: https://docs.aws.amazon.com/goto/WebAPI/apigateway-2015-07-09/CreateApiKey
* works (but failed when I tested friday): https://docs.aws.amazon.com/goto/WebAPI/cloudformation-2010-05-15/CancelUpdateStack
* works: https://docs.aws.amazon.com/goto/WebAPI/ec2-2016-11-15/AcceptReservedInstancesExchangeQuote

I've got a patch to insert appropriate docs.aws/goto links to the PAWS documentation for API docs, but that only works for some (majority?) of aws targets.   Similarly, I wanted to link each top level topics with the userguide, but even that is inconsistent as some have a developer guide instead of userguide and the urls for userguide randomly use /userguide/ and /ug/.

I don't really want to put in lmgtfy links. :(
